### DISCUSSION TITLE
Support for complex requirements sets?

### DISCUSSION THREAD

**Discussion by JDRay42** [kuafuai/DevOpsGPT#25] [Ideas]
In enterprise software development, requirements are complex and are typically arranged in a tiered structure (e.g. Epics, Features, and User Stories to use Agile terminology).  Is there currently (or planned) support for complex requirements sets?  It’s reasonable to think that a structured JSON file could be used as the transport mechanism, but the ability to connect to a system like Jira or Azure DevOps to retrieve and act on a project’s requirements would be an excellent feature.

**Comment by booboosui** [comment]
Good question, we do have plans to support the synchronization of requirements management tasks such as jira into DevOpsGPT for better integration with existing tools
